Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -102,6 +102,31 @@
     'ctph': 'file',
     'ssdeep': 'file'
 }
+
+
+class IndicatorTypes(object):
```

Comment:
I think maybe you should change the name of your class to `FeedIndicatorType` and I will change to `DBotScoreIndicatorType` this way it will be clear which class used where.